Makefile

```diff
@@ -246,6 +246,14 @@ kind-init-ovn-ic: kind-clean-ovn-ic kind-init-ha
 	@ha=true $(MAKE) kind-generate-config
 	$(call kind_create_cluster,yamls/kind.yaml,kube-ovn1)
 
+
+.PHONY: kind-init-ovn-submariner
+kind-init-ovn-submariner: kind-clean-ovn-submariner
+	@ha=true pod_cidr_v4=10.16.0.0/16 svc_cidr_v4=10.96.0.0/16 $(MAKE) kind-generate-config
```

Comment:
Does submariner (or the test cases) require clusters with more than 2 nodes?

Reply:
I'm not sure, the origin test script by lut777 has "ha=true", it may have some cases use it.